### PR TITLE
Fix/228/review readme md

### PR DESCRIPTION
**Pull request by algooapy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ With Tracim, you centralize information, you can stay in touch by configuring yo
 
 ### Manage documents and files ###
 
-Traceability and versionning are very important for high-quality processes. Unfortunately, specialized software are hard to setup and to use.
+Traceability and versioning are very important for high-quality processes. Unfortunately, specialized software are hard to setup and to use.
```
Comment (Skylsmoi) [inline README.md]: I think it's better to say "to setup and use" instead of "to setup and to use"

File: doc/setting.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+You may need an administrator account to manage Tracim. Use the following command (from ``/install/dir/of/tracim/tracim``):
+
+    gearbox user create -l admin@admin.admin -p admin@admin.admin -g managers -g administrators
+
+Keep in mind `admin-email@domain.com` must match with LDAP user.
+
+## Other parameters  ##
+
+There are other parameters which may be of some interest for you. For example, you can:
```
Comment (Skylsmoi) [inline doc/setting.md]: more like "which may be of interest TO you"

File: doc/database.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+
+# Database #
+
+## PostgreSQL ##
+
+If you want use `PostgreSQL` as database engine:
```
Comment (tracim) [inline doc/database.md]: If you want to use

File: doc/database.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+#### Creating a user and associated database ####
+
+You need a database and associated user/password. Tracim comes with `pgtool`, tool that will make this step easy:
+
+    ./bin/pgtool help
+
+Run the following self explanatory commands:
+
+    sudo su postgres
+    ./bin/pgtool create_user tracimuser tracimpassword
```
Comment (tracim) [inline doc/database.md]: Add chevrons over parameters to be changed by the user : 
```./bin/pgtool create_user <tracimuser> <tracimpassword>
./bin/pgtool create_database <tracimdb>
./bin/pgtool grant_all_privileges <tracimdb> <tracimuser>
```

File: doc/database.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    ./bin/pgtool create_database tracimdb
+    ./bin/pgtool grant_all_privileges tracimdb tracimuser
+    exit
+
+#### Test the database access ####
+
+So, now you have a database and an associated user/password. A good habit is to test things before to use them, that's why we want to test the database access. This is easily done with Tracim `pgtool`:
+
+    ./bin/pgtool test_connection tracimdb tracimuser tracimpassword 127.0.0.1
```
Comment (tracim) [inline doc/database.md]: same thing here : add chevrons

File: doc/database.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+    sudo apt install postgresql-server-dev-all postgresql postgresql-client
+
+### Minimalist introduction to PostgreSQL ###
+
+If you already use/know `PostgreSQL`, you can directly go to *Test the database access*.
+
+#### Driver ####
+
+Tracim uses the `psycopg` driver between the `SQLAlchemy` ORM and the `PostgreSQL` RDBMS. Run the following command to install the right version:
```
Comment (tracim) [inline doc/database.md]: psycopg => psycopg2

File: README.md [hunk 7]
```diff
@@ -107,292 +107,73 @@ Following the installation documentation below, you'll be able to run your own i
 
 # Installation #
 
-## Dependencies ##
+## Debian dependencies ##
```
Comment (tracim) [inline README.md]: Writing Debian dependencies excludes even before read other users. The original formulation allow other distribution OS not to feel excluded.

File: README.md [hunk 8]
```diff
@@ -107,292 +107,73 @@ Following the installation documentation below, you'll be able to run your own i
...
 
     source tg2env/bin/activate
 
-To install tracim and it's dependencies:
+Install Tracim and its dependencies:
 
     cd tracim && python setup.py develop && cd -
     pip install -r install/requirements.txt
 
-**Note**: If you want to use MySQL database, please refer to Configuration/database schema note to install required package.
```
Comment (tracim) [inline README.md]: You moved database documentation in another markdown file but it is not visible on the readme how to configure mysql or postgresql storage. Please add links to the specific documentations (something like "if you want to use mysql, please look at specific documentation - <here link to the documentation>)

Same thing for postgresql
Same thing for apache/wsgi configuration. 
Same thing for configuration parameters explanation

File: doc/database.md (hunk 4 above)
Comment (algooapy) [inline doc/database.md]: The database documentation page presents, as is, one main benefit:
- the ease for the reader to reproduce the command by simply copying/pasting the considered lines.

However, we should warn our reader to use this commands only for a development environment.

File: doc/database.md (hunk 5 above)
Comment (algooapy) [inline doc/database.md]: see above comment

**Pull request merged**